app/lib/methods/helpers/normalizeMessage.js

```diff
@@ -21,6 +21,11 @@ export default (msg) => {
 	if (!msg) {
 		return null;
 	}
+
```

Comment:
Try only removing `normalizeMessage` [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/lib/methods/helpers/mergeSubscriptionsRooms.js#L16) instead of this changes.

Reply:
Done! works so well.

![20200406_132154](https://user-images.githubusercontent.com/37127457/78582078-23ded180-780b-11ea-8f78-5138a452283b.gif)